### ISSUE TITLE
NgbRadioGroup keyboard navigation on Firefox is broken if focus is given using the mouse

### ISSUE THREAD

**Issue by jnizet** [ng-bootstrap/ng-bootstrap#1705]
### Bug description:

On Firefox (on MacOS at least), if you click a button of a NgbRadioGroup with the mouse and try to navigate using arrow keys, the focus doesn't move to the other radio buttons. It only works if you navigate to the first button of the group using the keyboard.

### Link to minimally-working plunker that reproduces the issue:

http://plnkr.co/edit/f1pkF2SdDYHZRANjJjHj?p=preview

### Version of Angular, ng-bootstrap, and Bootstrap:

Angular: 4.3.1

ng-bootstrap: 1.0.0-alpha.29

Bootstrap: 4.0.0-alpha.6

**Comment by pkozlowski-opensource** [comment]
Same happens on Safari (OSX) :-/

**Comment by pkozlowski-opensource** [comment]
Hmm, sounds like it might be on purpose (?): https://bugzilla.mozilla.org/show_bug.cgi?id=756028

**Comment by pkozlowski-opensource** [comment]
OK, seems like this is "by design" in both in FFox and Safari on OSX, here is the relevant discussion: https://bugzilla.mozilla.org/show_bug.cgi?id=577316

If you ask me I agree with the opener that it "feels" broken. But the bottom line is that this should be argued with FFox / Safari folks and not here. The sad reality is that focus / blur events are not fired for radio / checkboxes and we can't do much about it...

Closing since it "works as browser vendors decided it to work..."
